blobs/quickstarts/TypeScript/V12/nodejs/package.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "blob-quickstart-v12",

```

Comment:
We don't typically use the 'v12' notation anymore to reduce confusion. It's fine if it's in a file path, but we can probably remove references to v12 from the sample.

Reply:
Removed that and made another minor fix.